HEIF FAQ (#5192)
cloudflare/cloudflare-docs: content/images/image-resizing/format-limitations.md

Cloudflare Image Resizing can:

- Read JPEG, PNG, GIF (including animations), and WebP images.
    * SVG is not supported, since this format is inherently scalable and does not need resizing.

>     * We do not plan to support the HEIC (HEIF) format.

Reviewer: ```suggestion
    * Cloudflare does not plan to support the HEIC (HEIF) format.
```
